avoid including make.inc, which contains hardcoding settings we don't need/want
author: J-M Beuken
diff -Nru wannier90-3.0.0.orig/src/Makefile.2 wannier90-3.0.0/src/Makefile.2
--- wannier90-3.0.0.orig/src/Makefile.2	2019-06-26 22:47:08.067494977 +0200
+++ wannier90-3.0.0/src/Makefile.2	2019-06-26 22:47:35.313193842 +0200
@@ -2,7 +2,7 @@
 #  Should be no need to change below this line
 #
 
-include ../../make.inc
+#include ../../make.inc
 
 # Contains definition of OBJS, OBJSLIB, OBJS_POST, LIBRARY, DYNLIBRARY
 include ../Makefile.header
diff -Nru wannier90-3.0.0.orig/utility/w90pov/Makefile wannier90-3.0.0/utility/w90pov/Makefile
--- wannier90-3.0.0.orig/utility/w90pov/Makefile	2019-06-26 22:47:08.148494082 +0200
+++ wannier90-3.0.0/utility/w90pov/Makefile	2019-06-26 23:02:34.442673824 +0200
@@ -1,4 +1,4 @@
-include ../../make.inc
+#include ../../make.inc
 
 SRC=src
 OBJ=obj
diff -Nru wannier90-3.0.0.orig/utility/w90vdw/Makefile wannier90-3.0.0/utility/w90vdw/Makefile
--- wannier90-3.0.0.orig/utility/w90vdw/Makefile	2019-06-26 22:47:08.153494027 +0200
+++ wannier90-3.0.0/utility/w90vdw/Makefile	2019-06-26 23:03:01.118385092 +0200
@@ -1,4 +1,4 @@
-include ../../make.inc
+#include ../../make.inc
 
 w90vdw.x: w90vdw.f90
 	$(F90) $(FCOPTS) $< -o $@
